apps/src/netsim/NetSimSendPanel.js

```diff
@@ -74,6 +74,13 @@ var NetSimSendPanel = module.exports = function (rootDiv, levelConfig,
   this.packets_ = [];
 
   /**
+   * List of controllers for packets currently being edited.
+   * @type {NetSimPacketEditor[]}
+   * @private
+   */
+  this.lastPacketToAddress_ = 0;
```

Comment:
The documented type and default value here both seem to be wrong.  Is the comment a copy-paste error?  Also, addresses should always be treated as strings, since we support formats like "0.0.4" depending on the level; `"0"` might be a better default value here.